AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -3496,7 +3496,7 @@ legionathletics.com##.post-container > div#sign-up
 ||currenttime.tv/a/signup-sidebar-widget/
 ||d3olbu2cl74039.cloudfront.net/a/signup-sidebar-widget/
 ||d29z1quku5t3sk.cloudfront.net/a/signup-sidebar-widget/
-d3olbu2cl74039.cloudfront.net,d29z1quku5t3sk.cloudfront.net,currenttime.tv##.banner
+senior.ua,d3olbu2cl74039.cloudfront.net,d29z1quku5t3sk.cloudfront.net,currenttime.tv##.banner
```

Comment:
This is not a subscription. Used to create a profile on the site.
Added to self-promo:
`senior.ua##.register-with-button`